docs/index.md

```diff
@@ -0,0 +1,35 @@
+# Testcontainers
+
+![Testcontainers Banner](banner.png)
+
+## About
+
+Testcontainers for .NET is a library to support tests with throwaway instances of Docker containers for all compatible .NET Standard versions. The library is built on top of the .NET Docker remote API and provides a lightweight implementation to support your test environment in all circumstances.
+
+Choose from existing pre-configured configurations and start containers within a second, to support and run your tests. Or create your own container images with Dockerfiles and run your containers and tests immediately afterward.
+
+## Supported operating systems
+
+Testcontainers supports Windows, Linux, and macOS as host systems. Linux Docker containers are supported on all three operating systems. Native Windows Docker containers are only supported on Windows. Windows requires the host operating system version to match the container operating system version. You will find further information about Windows container version compatibility [here][windows-container-version-compatibility].
```

Comment:
I would say that Windows container support is currently limited (e.g. we are missing the Ryuk support, maybe that is the only thing missing, but it is generally an important feature across our TC messaging).

Reply:
The Ryuk support is covered in Reasource Reaper. For Windows we have a similar feature in .NET. As long as the test host does not crash, Testcontainesr for .NET will clean up Docker resources anyway.

Reply:
Yes, we have some basic tests here: https://github.com/testcontainers/testcontainers-dotnet/actions/runs/3329170074/jobs/5506046191